### ISSUE TITLE
[Feature] <title>Show some sort of picture like a key on a person's ID circle, to show that they have Session password-protected on their end

### ISSUE THREAD

**Issue by as-muncher** [oxen-io/session-desktop#3264]
### Is there an existing request for feature?

- [X] I have searched the existing issues

### What feature would you like?

I would like to see if a person has password-protected Session on his end.  That way I know that it is more secure for our conversations.  What good is all this encryption if the front door is unlocked?

**Comment by KeeJef** [comment]
There needs to be a balance between user privacy and signaling. I think revealing information about how other clients are configured by indicating that they have a password set might lean a little too heavily on signaling over user privacy.

**Comment by as-muncher** [comment]
@KeeJef If someone's Session is configured in a rather insecure way, that affects my security too.  I can't come out and ask someone if they have their Session password-protected: it's too easy to just say yes or no.

**Comment by keybreak** [comment]
See....Password protection is a **possibility** for security problem, however there are much more significant threats that you **should** be warned about, but clearly never will, because it will ScArE uSeRS on reality of our situation.
- [Microsoft Pluton](https://forum.endeavouros.com/t/microsoft-pluton-integrated-security-intel-amd-qualcomm/9784) is a chip-to-cloud backdoor inside pretty much 100% of all modern desktop and mobile CPUs.
Which means that you can **NOT** guarantee safety of your contacts, unless you personally know for a fact that contact and yourself are using older CPU released before 17 November 2020, RISC-V, or if it's mobile Google Pixel with GrapheneOS installed.
- Microsoft Windows 11 - is a 100% guarantee that all of your communications will be spied on through AI.
- Apple MacOS - is a 100% guarantee that all of your communications will be spied on through AI.

----------------------

tl;dr even if you're using older CPU desktop with Linux and Google Pixel with GrapheneOS, like myself - it's nearly impossible that any of your contact will, so assume your communications are not safe by default, regardless of which messenger is used .

**Comment by n-c0de-r** [comment]
Sorry. By that logic and assumptions (which sadly are true) we can drop session development & usage, as nothing is really safe anymore anyhow. Why bother trying for privacy if all is compromised to begin with in hardware level?
The argument goes against the app itself, in a way. But yeah, people should be informed about bigger threats - and a simple signal can make them more aware about their usage (if switching to Session didn't help enough).

**Comment by as-muncher** [comment]
What happened to @n-c0de-r's post?

**Comment by keybreak** [comment]
https://xcancel.com/session_app/status/1882244454259859535#m

Interesting, i wouldn't hold my breath...but yeah - he's on point.
If by some miracle Session would be first messenger to implement such warnings on conversation parties (which is great, because you wouldn't need to just **blindly** trust your partner), maybe your idea could be added to that list too.
